src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -448,68 +448,68 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       contents.description = (
         <p>The Veterans Benefits Administration agreed with you and decided to overturn the
         original decision. If this decision changes your disability rating or eligibility for VA
-        benefits, you should expect this adjustment to be made in 1 to 2 months.</p>
+        benefits, you should see this change made in 1 to 2 months.</p>
       );
       break;
     case STATUS_TYPES.withdrawn:
       contents.title = 'You withdrew your appeal';
       contents.description = (
-        <p>You have opted to not continue your appeal. If this information is incorrect, please
-        contact your Veteran Service Organization or representative for more information.</p>
+        <p>You have decided not to continue your appeal. If this information is incorrect, please
+        contact your Veterans Service Organization or representative for more information.</p>
       );
       break;
     case STATUS_TYPES.ftr:
-      contents.title = 'Your appeal was closed';
+      contents.title = 'We closed your appeal';
       contents.description = (
-        <p>You did not take an action VA requested in order to continue your appeal. If this
-        information is incorrect, or if you want to reopen your appeal, please contact your Veteran
+        <p>You didn’t take action on something we requested that would have kept your appeal open. If this
```

Comment:
Should be:

You did not take an action VA requested in order to continue your appeal. If this information is incorrect, or if you want to reopen your appeal, please contact your Veterans ...